### ISSUE TITLE
calendar@ccprog: UTC time zone missing in World Clocks

### ISSUE THREAD

**Issue by sphh** [linuxmint/cinnamon-spices-applets#3314]
```
 * Applet version/Build date: 1.1.3 (2020-11-16 20:54:10)
 * Cinnamon version: Cinnamon 4.6.7
 * Distribution: Linux Mint 20
 ```

**Notify author of applet**
@ccprog 

**Issue**
I want to add the UTC time zone to my World Clocks, but cannot find an appropriate entry. Where would I find it?

Thank you for your help!

**Comment by ccprog** [comment]
This is actually an upstream issue with [pytz](https://pypi.org/project/pytz/).

**Comment by sphh** [comment]
Thanks for looking into this.

I do not understand, why this is an upstream issue:
```python
>>> from pytz import common_timezones
>>> 'UTC' in all_timezones
True
```
It looks, like there are more timezones without a category (it's not a continent!) prepended:
```python
>>> from pytz import common_timezones
>>> [tz for tz in all_timezones if '/' not in tz]
['CET', 'CST6CDT', 'Cuba', 'EET', 'EST', 'EST5EDT', 'Egypt', 'Eire', 'GB', 'GB-Eire', 'GMT', 'GMT+0', 'GMT-0', 'GMT0', 'Greenwich', 'HST', 'Hongkong', 'Iceland', 'Iran', 'Israel', 'Jamaica', 'Japan', 'Kwajalein', 'Libya', 'MET', 'MST', 'MST7MDT', 'NZ', 'NZ-CHAT', 'Navajo', 'PRC', 'PST8PDT', 'Poland', 'Portugal', 'ROC', 'ROK', 'Singapore', 'Turkey', 'UCT', 'UTC', 'Universal', 'W-SU', 'WET', 'Zulu']
```
could they be summarized under the "continent" `Weird and wonderful` (or simply `Miscellaneous`? I just created PR #3322  implementing this. You might want to translate the variable `TZ_NO_CATEGORY` though.

**Comment by sphh** [comment]
Implemented by PR #3322
